### PR TITLE
Update helpers functions in subcharts

### PR DESCRIPTION
**Pull request by xpuska513**
## What
* Update the `component.imageTag` function and function call in all sub-charts to allow to use of both `global.image.tag` and `.image.tag` simultaneously.

### CODE REVIEW

File: charts/airbyte-server/templates/_helpers.tpl [hunk 1]
```diff
@@ -63,10 +63,10 @@ Define imageTag
 */}}
 
 {{- define "server.imageTag" -}}
-{{- if ((.Values.global.image).tag) }}
-    {{- printf "%s" .Values.global.image.tag }}
 {{- else if .Values.image.tag }}
```
Comment (cuyk) [inline charts/airbyte-server/templates/_helpers.tpl:66]:
```suggestion
{{- if .Values.image.tag }}
```

File: charts/airbyte-webapp/templates/_helpers.tpl [hunk 2]
```diff
@@ -64,10 +64,10 @@ Define imageTag
 */}}
 
 {{- define "webapp.imageTag" -}}
-{{- if ((.Values.global.image).tag) }}
-    {{- printf "%s" .Values.global.image.tag }}
 {{- else if .Values.image.tag }}
```
Comment (cuyk) [inline charts/airbyte-webapp/templates/_helpers.tpl:67]:
```suggestion
{{- if .Values.image.tag }}
```

File: charts/airbyte-worker/templates/_helpers.tpl [hunk 3]
```diff
@@ -63,10 +63,10 @@ Define imageTag
 */}}
 
 {{- define "worker.imageTag" -}}
-{{- if ((.Values.global.image).tag) }}
-    {{- printf "%s" .Values.global.image.tag }}
 {{- else if .Values.image.tag }}
```
Comment (cuyk) [inline charts/airbyte-worker/templates/_helpers.tpl:66]:
```suggestion
{{- if .Values.image.tag }}
```

**Review by cuyk — CHANGES_REQUESTED** [review]

File: charts/airbyte-server/templates/_helpers.tpl (hunk 1 above)
Comment (xpuska513) [reply charts/airbyte-server/templates/_helpers.tpl:66]: It doesn't work as expected, sample output:
`image: airbyte/bootloader:test2test`

Comment (xpuska513) [reply charts/airbyte-server/templates/_helpers.tpl:66]: Values.yaml reference:
```
##  global.serviceAccountName Name of service account to be associated with service
##  deploymentMode Determines deployment mode of airbyte
##  secretName Overrides the secrate name with with credentials to S3 Bucket and logging
##  database.secretName Name of database secret
##  database.secretValue Value of database password key stored in secret
##  database.host Hostname of database
##  database.port Database port
global:
  serviceAccountName: placeholderServiceAccount
  deploymentMode: oss
  secretName: ""
  database:
    secretName: ""
    secretValue: ""
    host: "example.com"
    port: "5432"
  secrets: {}
  env_vars: {}
  extraContainers: []
  image:
    tag: test


enabled: true
##  image.repository The repository to use for the airbyte bootloader image.
##  image.pullPolicy the pull policy to use for the airbyte bootloader image
##  image.tag The airbyte bootloader image tag. Defaults to the chart's AppVersion
image:
  repository: airbyte/bootloader
  pullPolicy: IfNotPresent
  tag: test2
```

File: charts/airbyte-webapp/templates/_helpers.tpl (hunk 2 above)
Comment (xpuska513) [reply charts/airbyte-webapp/templates/_helpers.tpl]: fixed

File: charts/airbyte-server/templates/_helpers.tpl (hunk 1 above)
Comment (xpuska513) [reply charts/airbyte-server/templates/_helpers.tpl]: Sorry, my mistake, fixed the typo, pushed changes to this branch.

**Review by cuyk — APPROVED** [review]

**Pull request merged**